assets/app/views/directives/_truncate.html

```diff
@@ -0,0 +1 @@
+<span ng-attr-title="{{content}}">{{content | limitTo : limit}}<span ng-if="content.length > (limit | toInt)">&hellip;</span></span>
```

Comment:
is there an easy way to see if truncating +/- 10ish characters would let us avoid chopping a word in half?